c/build.c

```diff
@@ -241,7 +241,7 @@ void save_to_file(kuliya_schema *schema, const size_t slots_length, const char *
         kuliyas_with_terms[kuliya_with_terms_idx].__varname = malloc(varname_length + 1);
         memcpy(kuliyas_with_terms[kuliya_with_terms_idx].__varname, path_value, varname_length);
         kuliyas_with_terms[kuliya_with_terms_idx]
-            .__varname[varname_length + 1] = '\0';
+            .__varname[varname_length] = '\0';
```

Comment:
interesting this was not caught on macOS

Reply:
hmm, could it be compiler related? are we using the same compilers in Linux and macOS?

Reply:
you're right, I was using clang on my system, and the action was probably using gcc